### PR TITLE
Add api doc for ng_style directive

### PR DESCRIPTION
**Pull request by jbogarthyde**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information
JIRA https://angular-team.atlassian.net/browse/DOCS-541

### CODE REVIEW

**Review by petebacondarwin — CHANGES_REQUESTED** [review]
The added file at `packages/common/src/ng_style.ts` should be deleted.

**Comment by jbogarthyde** [comment]
@petebacondarwin changes made, please remove changes-required status
@IgorMinar please approve/run g3 if needed

**Review by petebacondarwin — APPROVED** [review]

**Comment by brandonroberts** [comment]
@jbogarthyde I think its worth noting that the expressions have to evaluate a non-null value for the style to be applied. If an expression returns to a null value, the corresponding style is removed.

File: packages/common/src/directives/ng_style.ts [hunk 1]
```diff
@@ -12,21 +12,32 @@ import {Directive, DoCheck, ElementRef, Input, KeyValueChanges, KeyValueDiffer,
...
- * Update an HTML element styles.
- *
- * The styles are updated according to the value of the expression evaluation:
- * - keys are style names with an optional `.<unit>` suffix (ie 'top.px', 'font-style.em'),
- * - values are the values assigned to those properties (expressed in the given unit).
+ * An attribute directive that updates styles for the containing HTML element.
+ * Sets one or more style properties, specified as key-value pairs.
+ * The key is a style name, with an optional `.<unit>` suffix (such as 'top.px', 'font-style.em').
+ * The value is an expression to be evaluated.
+ * The resulting value, expressed in the given unit, is assigned to the given style property.
```
Comment (brandonroberts) [inline packages/common/src/directives/ng_style.ts]: I think its worth noting that the expressions have to evaluate a non-null value for the style to be applied. If an expression returns to a null value, the corresponding style is removed.

Comment (jbogarthyde) [reply packages/common/src/directives/ng_style.ts]: Done.

**Review by brandonroberts — CHANGES_REQUESTED** [review]
The inputs and public methods need descriptions. Otherwise, LGTM.

File: packages/common/src/directives/ng_style.ts [hunk 2]
```diff
@@ -41,13 +55,24 @@ export class NgStyle implements DoCheck {
       private _differs: KeyValueDiffers, private _ngEl: ElementRef, private _renderer: Renderer2) {}
 
   @Input()
-  set ngStyle(values: {[key: string]: string}) {
+  set ngStyle(
+      /**
+       * An array of style properties, specified as colon-separated
```
Comment (brandonroberts) [inline packages/common/src/directives/ng_style.ts]: This should be `An object` or `A map` instead of `An array`.

**Review by brandonroberts — CHANGES_REQUESTED** [review]

**Review by brandonroberts — APPROVED** [review]

**Comment by brandonroberts** [comment]
Caretaker note: This is a docs-only change that should not impact g3. Please run presubmit.

**Pull request closed without merging**